webapp/controller/OrderDescription.controller.js

```diff
@@ -0,0 +1,200 @@
+/*global location */
+sap.ui.define([
+	"./BaseController",
+	"sap/ui/model/json/JSONModel",
+	"../model/formatter",
+	"sap/m/library",
+		"sap/ui/core/routing/History"
+], function (BaseController, JSONModel, formatter, mobileLibrary, History) {
+	"use strict";
+
+	// shortcut for sap.m.URLHelper
+	var URLHelper = mobileLibrary.URLHelper;
+
+	return BaseController.extend("opensap.salesorderviewer.SalesOrderViewer.controller.OrderDescription", {
+
+		formatter: formatter,
+
+		/* =========================================================== */
+		/* lifecycle methods                                           */
+		/* =========================================================== */
+
+		onInit: function () {
+			// Model used to manipulate control states. The chosen values make sure,
+			// detail page is busy indication immediately so there is no break in
+			// between the busy indication for loading the view's meta data
+			var oViewModel = new JSONModel({
+				busy: false,
+				delay: 0
+			});
+
+			this.getRouter().getRoute("orderDescription").attachPatternMatched(this._onObjectMatched, this);

```

Comment:
wczesniej chyba tutaj w getRout() zapomnielismy zmienic na to jak sie nazywa ten rout u ciebie